envoy/api/v2/cds.proto

```diff
@@ -361,7 +361,19 @@ message Cluster {
     // Use HTTP1.1 or HTTP2, depending on which one is used on the downstream connection.
     USE_DOWNSTREAM_PROTOCOL = 1;
   }
+
+  // Determines how Envoy selects the protocol used to speak to upstream hosts.
   ClusterProtocolSelection protocol_selection = 26;
+
+  // Common settings for all load balancer implementations.
+  message CommonLoadBalancerSettings {
```

Comment:
Do you want to put this next to `lb_config` for organizational clarity? Also, should we keep the same naming convention, e.g. `common_lb_config`, `CommonLbConfig`. TBH, I prefer the naming scheme you have here, but consistency is powerful in an API.

Reply:
I can rename. Is our preference to keep things in order by field number? Or order logically? I thought we were trying to keep things in field order, but I'm happy to do logical ordering. We should codify either way in STYLE.md.

Reply:
Talked to Harvey offline. Will go with logical ordering. Will update.